assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -14,6 +15,7 @@ import './createResourceAccessChart.css'
 const accessedResourceColor = 'steelblue'
 const notAccessedResourceColor = 'gray'
 const mainMargin = { top: 50, right: 10, bottom: 50, left: 200 }
+const icons = {files: 'fas fa-file fa-lg', videos: 'fas fa-video fa-lg'}
```

Comment:
I think these icon names should be somehow added into the settings/config. Currently our RESOURCE_VALUES are like
```
    "RESOURCE_VALUES": {
        "files":  ["canvas"],
        "videos": ["leccap"]
    },
```
Maybe something like this?

```
    "RESOURCE_VALUES": {
        "files":  {"types": ["canvas"], "icons": "fas fa-file fa-lg"},
        "videos": ["leccap"], "icons": "fas fa-video fa-lg"}
    },
```

Then we can add new types. Maybe it doesn't need "fas fa-file fa-lg" but only just "fa-file"